dspace-api/src/main/java/org/dspace/app/bulkedit/MetadataImport.java

```diff
@@ -396,6 +427,19 @@ public MetadataImport(Context c, DSpaceCSV toImport) {
         return changes;
     }
 
+    private void handleRelationshipMetadataValueFromBulkEditMetadataValue(Item item, BulkEditMetadataValue dcv)
```

Comment:
While all the new methods in this class are `private`, I'd really appreciate either JavaDocs or inline code comments in these methods -- otherwise it becomes really complex to debug Bulk metadata import issues related to relationships.   I see a lot of new code added to this class, and zero additional comments.

Reply:
We’ll add some docs to these private methods as well

Reply:
This should be resolved using commit b37d0e216718316ba202f34556f53344cd0346b8